src/libcollections/vec_deque.rs

```diff
@@ -1243,14 +1242,19 @@ impl<T> VecDeque<T> {
     /// Panics if `index` is greater than `VecDeque`'s length
     ///
     /// # Examples
+    ///
     /// ```
     /// use std::collections::VecDeque;
     ///
-    /// let mut buf = VecDeque::new();
-    /// buf.push_back(10);
-    /// buf.push_back(12);
-    /// buf.insert(1, 11);
-    /// assert_eq!(Some(&11), buf.get(1));
+    /// let mut vec_deque = VecDeque::new();
+    /// vec_deque.push_back('a');
+    /// vec_deque.push_back('b');
+    /// vec_deque.push_back('c');
+    ///
+    /// vec_deque.insert(1, 'd');
+    ///
+    /// let vec = vec_deque.iter().collect::<Vec<_>>();
+    /// assert_eq!(vec, [&'a', &'d', &'b', &'c']);
```

Comment:
That didn't work for me:

```
error[E0277]: the trait bound `&char: std::cmp::PartialEq<char>` is not satisfied
  --> <anon>:12:4
   |
12 |    assert_eq!(vec, &['a', 'd', 'b', 'c']);
   |    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::cmp::PartialEq<char>` is not implemented for `&char`
   |
   = help: the following implementations were found:
   = help:   <char as std::cmp::PartialEq>
   = note: required because of the requirements on the impl of `std::cmp::PartialEq<&[char; 4]>` for `std::vec::Vec<&char>`
   = note: this error originates in a macro outside of the current crate

error: aborting due to previous error
```